x/oracle/keeper/crossChainFee_test.go

```diff
@@ -0,0 +1,29 @@
+package keeper_test
```

Comment:
Could we have this file test this function: `GetCrossChainFeeConfig`?

Reply:
We should assert both the happy and sad path for this function in unit and integration tests.

Reply:
will add both tests in unit testing.